src/types/express.d.ts

```
@@ -1,9 +1,22 @@
     user?: Partial<User>;
     validatedIp?: string;
     tokenExp?: number;
+
+    // adding this for  audit
+    auditContext?: {
+      userId: string;
+      userEmail: string;
+      ipAddress: string;
+      userAgent: string;
+    };
+    preAuditEntity?: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)